pkg/printer/printer.go

```diff
@@ -384,6 +384,170 @@ func (p *Printer) WriteProtoNodeStatusEvent(r *observerpb.GetFlowsResponse) erro
 	return nil
 }
 
+func formatServiceAddr(a *pb.ServiceUpsertNotificationAddr) string {
+	return fmt.Sprintf("%s:%d", a.Ip, a.Port)
```

Comment:
Please, use [`net.JoinHostPort`](https://golang.org/pkg/net/#JoinHostPort) so that IPv6 addresses are properly supported.

Reply:
Thanks for catching this.

Reply:
🤦 I addressed this locally but forget to amend and push before I clicked the merge button. Sorry about that 😞 will send a follow-up PR.

Reply:
Addressed in #454